cuqi/sampler/_conjugate.py

```diff
@@ -0,0 +1,37 @@
+from cuqi.distribution import Posterior, GaussianCov, GaussianPrec, Gamma, GMRF
+import numpy as np
+
+class Conjugate:
```

Comment:
See below comment under Approximate conjugate sampler, which applies to both cases.